consensus/consensus.go

```diff
@@ -65,6 +65,7 @@ type ConsensusParams struct {
 	Logger         hclog.Logger
 	Metrics        *Metrics
 	SecretsManager secrets.SecretsManager
+	BlockTime      uint64
```

Comment:
comment indicating it is ms

Reply:
Added comment 631515404cf370d1fc015938ea35da52ff9dbaaf